Story/Episode 5/st_056120.csv

```diff
@@ -0,0 +1,34 @@
+st_056120_0010#0,"""All across the Kingdom of Epic, mysterious Devil Castles<br>began popping up without any trace of a summoner."""
+st_056120_0020#0,"""While the citizens had all been evacuated<br>it would all be for naught if their livelihoods<br>were obliterated by the castles."""
+st_056120_0030#0,"""Thus, the Guardian's group advanced through Epic<br>destroying Devil Castles whenever they came across one."""
+st_056120_0040#0,"""Their goal was to defeat the four that had taken over Epic<br>Each of the four had taken one of the four forts in the<br>four corners of Epic, which served originally as government centers."""
+st_056120_0050#0,"""To the Eastern Fortress lay<br>one such mighty foe."""
+st_056120_0060#0,"""The Guardian pressed onward towards their foe but,<br>they were about to meet with a cruel twist of fate..."""
+st_056120_0080#0,"""Ah, There! Her! She's one of the four we're after!"""
+st_056120_0100#0,"""Hm just as I predicted, you've arrived at last. Guardian!"""
+st_056120_0110#0,"""Kyukyukyu, kyu!"""
+st_056120_0130#0,"""Uwaa..."""
+st_056120_0150#0,"""Uwaa..."""
+st_056120_0170#0,"""...It seems as though my presence has cowed you into silence."""
+st_056120_0180#0,"""No, we're just extremely confused!"""
+st_056120_0190#0,"""Then, fear not. For there is ample room to rally beneath my banner."""
+st_056120_0200#0,"""I have heard tale of your deeds many, many times from the Great Matoi. So many that one could say my ears began to blister."""
+st_056120_0210#0,"""E-everything Aika's saying is all muddled up. There's even some stuff from Oracle mixed in there..."""
+st_056120_0220#0,"""There's signs of Aika in there, but I can sense a Dark Falz-like presence too..."""
+st_056120_0230#0,"""I do not know of her past, but there is no mistake that she is under the influence of Ephimera. Do not be deceived by her words, she is dangerous."""
+st_056120_0240#0,"""It seems as though our differences are irreconcilable. A pity that, now you will all have to perish."""
+st_056120_0250#0,"""Shut your mouth, foul traitor! You think we'll lie down and let you trample all over my country?! Think again, you're going down!"""
+st_056120_0270#0,"""...Aren't you getting a little too into this, Rita?"""
+st_056120_0280#0,"""Uh, well... It... Felt like one of those moments out of a heroic story, so I wanted to give it a try..."""
+st_056120_0290#0,"""You are the one who calls herself 'Queen' of this country, are you not? You were the one who summoned us forth, so why do you stand against us now."""
+st_056120_0300#0,"""We, the Four Infernal Lords of [Apprentice], are doing exactly as you asked, no?"""
```

Comment:
Now, you might shoot us for this, but we really wanted to push The `Elite Four` of Apprentice.

Reply:
Another friend suggested that, and I thought it wasn't chuuni enough. Especially when the point of the scene is to be overblown and dramatic. Plus, Elite Four Apprentices sounds a little silly and runs into the same problem as before, and Elite Four of [Apprentice] sounds a little ESL. I-I worked very hard on making sure it sounded the right way, I swear!